AUTHORS.rst

```diff
@@ -0,0 +1,8 @@
+Authors
+=======
+
+The list of contributors in alphabetical order:
+
+- Diego Rodriguez <diego.rodriguez@cern.ch>
```

Comment:
Please take ORCID links from `reana` repository.

BTW the author list would include only you for now, we have not contributed anything yet to this particular repository :wink: